scripts/choose-test-instance.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+INSTANCE_TESTCAFE="None"
+if [ "$TRAVIS_PULL_REQUEST" != "false" ];
+then
+ let INSTANCE_NUM=$TRAVIS_PULL_REQUEST%4;
+ INSTANCE_TESTCAFE="testcafe"$INSTANCE_NUM".cozy.rocks";
+fi
+echo "export INSTANCE_TESTCAFE=$INSTANCE_TESTCAFE" > setinstance.sh
```

Comment:
I think you can avoid using a temporary script by using `export INSTANCE_TESTCAFE="$INSTANCE_TESTCAFE"` here, and calling this script from `.travis.yml` like this `- . ./scripts/choose-test-instance.sh`. The `.` tells the shell to run the script inside the current process, instead of creating a sub-shell.